src/components/dropdowns-deprecated/DropdownDeprecated.jsx

```diff
@@ -0,0 +1,61 @@
+// @flow strict
+
+import React, {useEffect, useState, useRef} from 'react';
+import classNames from 'classnames';
+
+import Icon from '../icons/Icon';
+
+type LinksType = {
+  label: string,
+  url: string,
+};
+
+type PropsType = {
```

Comment:
```
type PropsType = $ReadOnly<{
  name: string,
  links: <LinksType>,
}>
```

all the props seem to be ReadOnly. wdyt?

Reply:
fixed